experimental/examples/bubblebench/cursor-shared-tree/src/schema.ts

```diff
@@ -0,0 +1,71 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import {
+    brand,
+    emptyField,
+    FieldKinds,
+    fieldSchema,
+    namedTreeSchema,
+    rootFieldKey,
+    SchemaData,
+    ValueSchema,
+} from "@fluid-internal/tree";
+
+export const stringSchema = namedTreeSchema({
+    name: brand("String"),
+    extraLocalFields: emptyField,
+    value: ValueSchema.String,
+});
+
+export const numberSchema = namedTreeSchema({
+    name: brand("number"),
+    value: ValueSchema.Number,
+});
+
+export const bubbleSchema = namedTreeSchema({
+    name: brand("Test:BubbleBenchAppStateBubble-1.0.0"),
+    localFields: {
+        x: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        y: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        r: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        vx: fieldSchema(FieldKinds.value, [numberSchema.name]),
+        vy: fieldSchema(FieldKinds.value, [numberSchema.name]),
+    },
+});
+
+export const clientSchema = namedTreeSchema({
+    name: brand("Test:BubbleBenchAppStateClient-1.0.0"),
+    localFields: {
+        clientId: fieldSchema(FieldKinds.value, [stringSchema.name]),
+        color: fieldSchema(FieldKinds.value, [stringSchema.name]),
+        bubbles: fieldSchema(FieldKinds.sequence, [bubbleSchema.name]),
+    },
+});
+
+
+export const appStateSchema = namedTreeSchema({
```

Comment:
We simplified the schema in the other version not to have this node.

It would be nice to share the schema between these two versions, as well as things like the initial trees and such.
Can we import the schema from the other version? If not we should at least do the same simplifications to this one.

Maybe it would work better to make the cursor and editiable tree versions in the same package, and just check some configuration in the top level index.ts file to see which mode we want to run it (which would pick the right app state).